analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.cs

```diff
@@ -117,5 +115,70 @@ public void Branching_PersistSymbols_BetweenBlocks()
             collector.ValidateTag("ElseFirst", x => x.HasConstraint(TestConstraint.First).Should().BeTrue());
             collector.ValidateTag("ElseSecond", x => x.HasConstraint(TestConstraint.Second).Should().BeTrue());
         }
+
+        [TestMethod]
+        public void ExitReached_SimpleFlow() =>
+            SETestContext.CreateCS("var a = true;").Collector.ValidateExitReachCount(1);
+
+        [TestMethod]
+        public void ExitReached_MultipleBranches()
+        {
+            const string method = @"
+public int Method(bool a)
+{
+    if (a)
+        return 1;
+    else
+        return 2;
+}";
+            SETestContext.CreateCSMethod(method).Collector.ValidateExitReachCount(1);
+        }
+
+        [TestMethod]
+        public void ExitReached_Throw()
+        {
+            const string method = @"
+public int Method(bool a)
+{
+    if (a)
+        throw new System.NullReferenceException();
+    else
+        return 2;
+}";
+            var returnNotReached = new PreProcessTestCheck(x =>
+                                                           {
+                                                               x.Operation.Instance.Kind.Should().NotBe(OperationKind.Literal, "we don't support multiple branches yet");
+                                                               return x.State;
+                                                           });
```

Comment:
The `{...}` lambda block should be aligned on 12 spaces.  Same as `, c=> ` in rule registrations. The convention of far-aligning on lambdas is for cases like
```
xxxxxxxxxxxxxxx ( x=> longconditioooooooooooooooooooooooooooooon
                                   && longconditioooooooooooooooooooon)
```